corehq/util/soft_assert/api.py

```diff
@@ -22,7 +22,8 @@ def _send_message(info, backend):
 
 
 def soft_assert(to=None, notify_admins=False,
-                fail_if_debug=False, exponential_backoff=True, skip_frames=0):
+                fail_if_debug=False, exponential_backoff=True, skip_frames=0,
+                send_to_ops=True,):
```

Comment:
ah, found the context on slack. I'm not 100% on board with `True` as a default. Seems like this could cause a lot of email spam for people with assertions that really only should belong to them (though I guess fine to force that to be explicit). Either way, I think this should be combined with `notify_admins` since they now basically provide two ways to do a very similar thing.

Longer term I really think we need an anomalous event recording system that isn't based on pushing emails to the entire ops team, but that's a separate and out-of-scope point.

@dimagi/team-commcare-hq better update your email filters for this!